src/functionalTest/java/uk/gov/hmcts/reform/dataextractor/ExtractorCsvFTest.java

```diff
@@ -12,7 +12,7 @@
 
 
 @Testcontainers
-public class ExtractorCsvTest extends DbTest {
+public class ExtractorCsvFTest extends DbTest {
```

Comment:
It seems pitest does not detect these files as the name is the same as unit test